### ISSUE TITLE
Importing a file into a table gives a "SELECT command denied to user ..." for table pma_tracking error

### ISSUE THREAD

**Issue by svaningelgem** [phpmyadmin/phpmyadmin#13214]
### Steps to reproduce
1. have in the config a controluser/controlpass with a user which has access to the pma database
1. Log in with a user which does NOT have access to this database
2. "Import" something in the database (I tried with a CSV, ";" delimited, "update values" checked).

### Expected behaviour
The import succeeds without errors

### Actual behaviour
I get an error "#1142 - SELECT command denied to user '<user>'@'server' for table 'pma_tracking'"

### Server configuration
**Operating system**: Debian GNU/Linux 8.7 (jessie)
**Web server:** Apache 2.4
**Database:** MySQL 5.6.36
**PHP version:** PHP 5.6.30-0+deb8u1
**phpMyAdmin version:** 4.7.0

### Client configuration
**Browser:** Chrome 57.0.2987.133
**Operating system:** Windows 7 64bit Home Premium

**Comment by udan11** [comment]
This is most probably related to configuration storage. Can you check that configuration and report if anything looks suspicious? [Here](https://docs.phpmyadmin.net/en/latest/setup.html#manual-configuration) you can find installation instruction about it.

**Comment by svaningelgem** [comment]
I've been using the config.inc.php like forever without any mayor changes.
I always nicely updated these tables as well (be aware that my tables are called pma_tracking (1 underscore) instead of the pma__tracking (2 underscores) used in the SQL files).

I don't have a ZeroConf-setting in my configuration file (but I also do not get a prompt to create the configuration).
In my config I do have these related settings:
```
$cfg['Servers'][$i]['controluser'] = 'pma';
$cfg['Servers'][$i]['controlpass'] = 'secret';

$cfg['Servers'][$i]['pmadb']            = 'phpmyadmin';
$cfg['Servers'][$i]['bookmarktable']    = 'pma_bookmark';
$cfg['Servers'][$i]['relation']         = 'pma_relation';
$cfg['Servers'][$i]['table_info']       = 'pma_table_info';
$cfg['Servers'][$i]['table_coords']     = 'pma_table_coords';
$cfg['Servers'][$i]['pdf_pages']        = 'pma_pdf_pages';
$cfg['Servers'][$i]['column_info']      = 'pma_column_info';
$cfg['Servers'][$i]['history']          = 'pma_history';
$cfg['Servers'][$i]['table_uiprefs']    = 'pma_table_uiprefs';
$cfg['Servers'][$i]['tracking']         = 'pma_tracking';
$cfg['Servers'][$i]['userconfig']       = 'pma_userconfig';
$cfg['Servers'][$i]['designer_coords']  = 'pma_designer_coords';
$cfg['Servers'][$i]['recent']           = 'pma_recent';
$cfg['Servers'][$i]['favorite']         = 'pma_favorite';
$cfg['Servers'][$i]['users']            = 'pma_users';
$cfg['Servers'][$i]['usergroups']       = 'pma_usergroups';
$cfg['Servers'][$i]['navigationhiding'] = 'pma_navigationhiding';
$cfg['Servers'][$i]['savedsearches']    = 'pma_savedsearches';
$cfg['Servers'][$i]['central_columns']  = 'pma_central_columns';
```

**Comment by svaningelgem** [comment]
The weird thing is that the data is imported correctly (it says "Import has been successfully finished, 21 queries executed. "). But it still gives the error:
![image](https://cloud.githubusercontent.com/assets/199434/25556646/4f9ee8fe-2d00-11e7-9903-a532d73902a7.png)

I tried debugging it, but couldn't really get far as I don't know exactly where this error comes from.
For example what I don't get:
In the picture you see clearly "Import has been successfully finished". This string is only referenced in "import.php, line 592. So I added in there:
```
  ob_start();
  echo "----------------\n";
  var_dump($finished, $error, $import_type, $executed_queries, $import_notice);
  file_put_contents('/tmp/import.txt', ob_get_contents(), FILE_APPEND);
  ob_end_clean();
```
But I don't get an import.txt file... Meaning the phpmyadmin code just doesn't pass there.
So I don't get it... How can it

**Comment by udan11** [comment]
Can you grant privileges for `pma` user again?

```sql
GRANT SELECT, INSERT, UPDATE, DELETE ON pma.* TO 'pma'@'localhost' IDENTIFIED BY 'pmapass';
```

**Comment by svaningelgem** [comment]
I did that on your request, but that didn't change anything.
I'm now digging into import.php. I am currently at line 714 (PMA_executeQueryAndGetQueryResponse) where it gets into, but never out.
The status at that moment is finished = true, query = table & error = false.

I'll keep you up-to-date.

**Comment by svaningelgem** [comment]
Found it!

```
    static public function isTracked($dbname, $tablename)
>>        $result = $GLOBALS['dbi']->fetchValue($sql_query);
```

You do the fetching here as the userlink (the default), not the controllink. Hence the error.
I don't really know what the proper way to handle this is, but this patch fixed the issue for me:
```
Index: Tracker.php
===================================================================
--- Tracker.php	(revision 4133)
+++ Tracker.php	(working copy)
@@ -127,7 +127,10 @@
         " AND table_name = '" . $GLOBALS['dbi']->escapeString($tablename) . "' " .
         " ORDER BY version DESC LIMIT 1";
 
-        $result = $GLOBALS['dbi']->fetchValue($sql_query);
+        $row = $GLOBALS['dbi']->fetchArray(PMA_queryAsControlUser($sql_query));
+        return isset($row[0])
+            ? $row[0]
+            : -1;
 
         return ($result == 1);
     }
```

**Comment by udan11** [comment]
Would you like submitting a pull request for this fix? If not, I can do it for you.

**Comment by svaningelgem** [comment]
Sorry, I don't have git at this side... Only SVN :-). Hence the patch I
created. I really ought to start having git :-)

**Comment by nijel** [comment]
Sorry guys to interrupt you, but this has been already fixed in cf525c78761e4a984163c975a418580a5a024043 and is duplicate of #13132 :-).
